src/modules/ROOT/pages/changelog/202209/08.json

```diff
@@ -0,0 +1,110 @@
+{
+  "meta": {
+    "releasedOn": "2023.03.01"
+  },
+  "added": [
+    {
+      "what": [
+        "configuration option that hides record list filtering."
+      ],
+      "why": [
+        "to enable configurators to hide filtering on record lists."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/8ebd25c4b1596af467578848fe9d4eb295d84c80"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "the link to the reminder record, it was added in the reminder sidebar list/edit.",
+        "Enabled the ability to un-dismiss reminders.",
+        "Added a date input field to set the date and time of the reminder.",
+        "General UI/UX improvements for reminders."
+      ],
+      "why": [
+        "to make reminders more user friendly and easier to use by having the option to set reminder in more than 24h and to have the option to link a record to a reminder and also to un-dismiss a reminder if needed."
+      ],
+      "how": [
+        "by updating the reminders logic and configuration in our code base."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/abc8ad4a5f5ebff0c9b21d9af9dd25cf1e0f2fba",
+        "https://github.com/cortezaproject/corteza/pull/899/commits/91a86acf75d11626fbec716dde1ba2f2305bbd1c",
+        "https://github.com/cortezaproject/corteza/pull/899/commits/5a559d2be90f54db1806110b0231d27e872b530f",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/6e1dca23d00d9d44ca1c7febd06d49b28de94d99",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/86452eed40176a83bbc24f12ebae66c0f3f3a04e",
+        "https://github.com/cortezaproject/corteza/pull/885/commits/178e64dc1ea190e42eed530eaccd791854c057f1"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "date multi value filtering using contains/not contains."
```

Comment:
Prefer `...using contains and not contains...` over `...using contains/not contains...`